onepane/README.md

```diff
@@ -0,0 +1,102 @@
+# Onepane
+
+## Overview
+
+Onepane is a GenAI tool that enhances incident resolution through automated Root Cause Analysis (RCA) by integrating seamlessly with monitoring tools like Datadog.
+By integrating Onepane with Datadog, customers gain rapid incident correlation with changes and infrastructure events, significantly reducing troubleshooting time and improving operational efficiency.
+The integration tracks incident data, infrastructure events, and change data (like code updates), providing valuable insights that help teams quickly identify root causes and prevent future issues.
+
+
+## Setup
+
+In Datadog:
+
+- Generate an Application Key:
+
+    - Access your personal settings in Datadog.
+
+    - Navigate to the Application keys section under Organization settings.
+
+    - Select "New Key" to generate a new Application key.
+
+    - Name the key and create it.
+
+    - Copy the generated Application key for future use.
+
+In Onepane:
+
+Here's how to connect your Datadog account to Onepane 
+
+- Access Onepane Console: Head over to the Onepane console at [console.onepane.ai][2]
+
+- Locate Integrations: In the left-hand menu, find the "Integrations" section. Click the "Add Integration" button in the top right corner.
+
+- Choose Datadog: From the list of available integrations, select "Datadog".
+
+- When you select Datadog, you will be redirected to the Datadog login page. 
+
+- Authenticate with OAuth.
+
+    - Log in using your Datadog credentials.
+
+    - Upon successful authentication, Datadog will create an API key for you.
+
+- Provide a name for your Datadog connector.
+
+- Input the site URL and Application key obtained from the ![prerequisite steps][8].
+
```

Comment:
OAuth must be intiated from Datadog. To make OAuth work for users across all Datadog regions, you will need to ensure that you’re making the correct API calls based on their region. **When the user kicks off authorization from the Datadog tile, a site parameter is sent on redirect from the onboarding_url. You will use this site parameter in your calls to the Authorize and Token endpoints. Please note that if a user kicks off authorization directly from your platform, this site parameter will not be sent, and the user will be prompted to select their site on the Datadog authorize page.**